judgels-frontends/raphael/src/routes/uriel/modules/urielReducer.ts

```diff
@@ -4,10 +4,12 @@ import storage from 'redux-persist/es/storage';
 
 import { contestReducer, ContestState } from '../contests/modules/contestReducer';
 import { contestWebConfigReducer, ContestWebConfigState } from '../contests/modules/contestWebConfigReducer';
+import { ContestDescriptionState } from '../contests/modules/contestDescriptionReducer';
 
 export interface UrielState {
   contest: ContestState;
   contestWebConfig: ContestWebConfigState;
+  contestDescription: ContestDescriptionState;
```

Comment:
You don't need to store the description in Redux store at all, because it is only used in contest overview page.

It is better to only fetch the description in that page.